mobile/src/libs/components/card/card.tsx

```diff
@@ -2,27 +2,42 @@ import React from 'react';
 import { type ImageSourcePropType } from 'react-native';
 
 import imagePlaceholder from '#assets/img/card-image-placeholder.png';
-import { Image, Text, TouchableOpacity } from '#libs/components/components';
+import {
+  Image,
+  Pressable,
+  Text,
+  TouchableOpacity,
+  View,
+} from '#libs/components/components';
 
 import { DEFAULT_NUMBER_OF_LINES } from './libs/constants';
 import { styles } from './styles';
 
 type Properties = {
   title: string;
   image?: ImageSourcePropType;
+  iconSourceSvg?: JSX.Element;
   onPress: (title: string) => void;
 };
 
 const Card: React.FC<Properties> = ({
   title,
   image = imagePlaceholder,
+  iconSourceSvg,
   onPress,
 }) => {
   const handlePress = (): void => {
     onPress(title);
   };
 
-  return (
+  return iconSourceSvg ? (
+    <Pressable onPress={handlePress}>
+      <View style={styles.container}>
+        <View style={styles.iconContainer}>{iconSourceSvg}</View>
+        <Text style={styles.title}>{title}</Text>
+      </View>
+    </Pressable>
+  ) : (
```

Comment:
Is this block really needed? you are reusing most of the styles so you can probably move the ternary operator inside the card block

Reply:
@Nanaier yeah it is needed , because this block has different structure compared to another one. and i tried to make it as you say, but layout looked badly